vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/MessageInformationDataFactory.kt

```diff
@@ -55,7 +57,7 @@ class MessageInformationDataFactory @Inject constructor(
         private val reactionsSummaryFactory: ReactionsSummaryFactory
 ) {
 
-    fun create(params: TimelineItemFactoryParams): MessageInformationData {
+    fun create(params: TimelineItemFactoryParams, lastEdit: Event? = null): MessageInformationData {
```

Comment:
I am not sure it is needed to add the `lastEdit` as a new argument. Since it is already contained in the first argument, I think we could retrieve it easily inside this method.

Reply:
good catch